.editorconfig

```diff
@@ -0,0 +1,12 @@
+root = true
```

Comment:
Actually it's needed, cause you are using windows `end_of_line`, I'm unix.
This file specify this. This file also helps other people that would to contribute.
http://editorconfig.org/